frontend/src/components/Pages/MissionPage/MissionHeader/MissionHeader.tsx

```diff
@@ -115,37 +151,61 @@ export const MissionHeader = ({ mission }: { mission: Mission }) => {
-                {!isMissionCompleted && HeaderText(`${translatedBatteryLevel}: ${mission.robot.batteryLevel}%`)}
-                {!isMissionCompleted &&
-                    mission.robot.pressureLevel &&
-                    HeaderText(
-                        `${translatedPressureLevel}: ${Math.round(mission.robot.pressureLevel * barToMillibar)}mBar`
+        <>
+            <HeaderSection>
+                <TitleSection>
+                    <StyledTypography>{mission.name}</StyledTypography>
+                    {isMissionActive && (
+                        <MissionControlButtons
+                            missionName={mission.name}
+                            robotId={mission.robot.id}
+                            missionStatus={mission.status}
+                        />
+                    )}
+                    {mission.endTime && (
+                        <MissionRestartButton mission={mission} hasFailedTasks={missionHasFailedTasks} />
                     )}
+                </TitleSection>
+                <Typography
+                    variant="body_long"
+                    group="paragraph"
+                    color={tokens.colors.text.static_icons__secondary.hex}
+                >
+                    {mission.description && `${translatedDescription}: ${mission.description}`}
+                </Typography>
+                <StatusReason statusReason={mission.statusReason} status={mission.status}></StatusReason>
+            </HeaderSection>
+            <InfoSection>
+                <StyledCard>
+                    <div>
+                        {HeaderText(translatedStatus, '')}
+                        <MissionStatusDisplay status={mission.status} />
+                    </div>
+                    {HeaderText(translatedArea, `${mission.area?.areaName}`)}
+                    {HeaderText(translatedTasks, `${numberOfCompletedTasks + '/' + mission.tasks.length}`)}
+                </StyledCard>
+                <StyledCard>
+                    {HeaderText(translatedStartDate, `${startDate}`)}
+                    {HeaderText(translatedStartTime, `${startTime}`)}
+                </StyledCard>
+                <StyledCard>
+                    {HeaderText(translatedUsedTime, `${usedTime}`)}
+                    {!isMissionCompleted && HeaderText(translatedEstimatedTimeRemaining, `${remainingTime}`)}
+                </StyledCard>
+                <StyledCard>
+                    {HeaderText(translatedRobot, `${mission.robot.name}`)}
+                    {!isMissionCompleted && HeaderText(translatedBatteryLevel, `${mission.robot.batteryLevel}%`)}
+                    {!isMissionCompleted &&
+                        mission.robot.pressureLevel &&
+                        HeaderText(
+                            translatedPressureLevel,
+                            `${Math.round(mission.robot.pressureLevel * barToMillibar)}mBar`
+                        )}
```

Comment:
Alignment is off on completed missions 
![image](https://github.com/equinor/flotilla/assets/33518988/bc956674-e269-4dfe-9e68-e5cb1f3ede6f)